systems/sensors/test/sim_cassie_sensor_aggregator_test.cc

```diff
@@ -0,0 +1,56 @@
+#include "systems/sensors/sim_cassie_sensor_aggregator.h"
```

Comment:
Is there an easy way to test that the results come back OK? Maybe confirm that the acceleration is 0 for the fixed base, and -g for the floating bases, and a similar test of non-zero angular velocity for the floating base models.

Reply:
Do we need to test the IMU value? The job of SimCassieSensorAggregator is just gathering the data from accelerometer and gyroscope and put them into lcmt_cassie_out.

Reply:
Fair enough.